src/components/molecules/NavBar/playa.scss

```diff
@@ -60,12 +59,13 @@ $sand: #937c63;
     }
 
     .navbar-links {
-      padding: 12px 20px;
+      padding: 11px 20px;
```

Comment:
Why the `1px` change here? And how are we calculating the 'magic number' `11`/etc? If it relates to some other sizes/etc elsewhere, can we use variables and 'show the workings' here?

Reply:
when the color to `.navbar-links` became not transparent, it turned out that it hides border-bottom of the `.navbar` - see the yellow line, so I just decreased it a bit.
![image](https://user-images.githubusercontent.com/63313732/116521801-6eae0a00-a8a2-11eb-82db-3469741c98b6.png)

Reply:
Makes sense. Is it possible for this element to be 'properly container' within the NavBar, rather than being able to 'break out of it' like this?

Reply:
I can't see what is the best way to do that right now.

In fact, it might be worth removing that padding fully. Not sure what was the original purpose of it.

with the padding:
![image](https://user-images.githubusercontent.com/63313732/116663884-b6499a00-a965-11eb-9f22-4789913adcc9.png)

without the padding: 
![image](https://user-images.githubusercontent.com/63313732/116664005-dd07d080-a965-11eb-8a98-89aa243df135.png)

removing padding, profile icon comes closer to the right side. 
And looking at the design http://sapiens.chat/sparkle/styleguide/map.html it seem to be closer to the desired result.

@mathieud do you think, we need to remove the padding?

Reply:
agreed with @mike-lvov to remove paddings. 

it broke the view in test env, for some reason.
![image](https://user-images.githubusercontent.com/63313732/116686454-d2f3cb00-a981-11eb-9bb6-d131373fd73c.png)

https://github.com/sparkletown/sparkle/pull/1350/commits/d5c1e7f70916dcf9d54798b9f636a1523eb8785a
https://github.com/sparkletown/sparkle/pull/1350/commits/ff3ef23aa0fda2796f98e07ea004a9170eb80ef6

Reply:
@sunny-viktoryia FYI, @mcflyok is Mathieu

Reply:
> it broke the view in test env, for some reason.

@sunny-viktoryia Is this still an outstanding issue? Or is it resolved now?